opennms-base-assembly/src/main/filtered/etc/alarmd/drools-rules.d/alarmd.drl

```diff
@@ -172,7 +172,6 @@ rule "closeClearedAlarmTickets"
     $sessionClock : SessionClock()
     $ticketer : AlarmTicketerService(isTicketingEnabled() == true)
     $alarm : OnmsAlarm(severity == OnmsSeverity.CLEARED &&
-                       alarmType == OnmsAlarm.PROBLEM_TYPE &&
```

Comment:
@j-white  This was needed for UseCase3 where `clearAlarm` should close open ticket after 15 mins. But when clear alarm comes, it seems like it has `OnmsAlarm.RESOLUTION_TYPE`.  I'm not sure why this check was there before. Once I removed this check, ticket gets closed and removed properly.  Let me know if I'm missing anything.

Reply:
Logic was copied from the old vacuumd statements: https://github.com/OpenNMS/opennms/blob/opennms-19.1.0-1/opennms-base-assembly/src/main/filtered/etc/vacuumd-configuration.xml#L185

The clear should be reduced on a problem - would need to take a closer look and see why this is not the case.

Reply:
Looks like this was coming from [here](https://github.com/OpenNMS/opennms/blame/bcd0b266b23c458fc30e1c4b47014ff462ae3bd7/opennms-alarms/daemon/src/main/java/org/opennms/netmgt/alarmd/AlarmPersisterImpl.java#L228).

Removing that doesn't resolve the issue. That actually `unclear` alarm by this [rule](https://github.com/OpenNMS/opennms/blame/bcd0b266b23c458fc30e1c4b47014ff462ae3bd7/opennms-alarms/daemon/src/main/java/org/opennms/netmgt/alarmd/AlarmPersisterImpl.java#L228) and ticket never gets closed.

I'm puzzled how it worked with `vacuumd-configuration` in previous versions.

Reply:
Looks like `DefaultAlarmTicketerService#createTicket`  updates `lastAutomationTime` and when clear event comes, it satisfies the condition for `unclear`.  Previous `vacuumd-configuration` doesn't seem to update `lastAutomationTime` which may be the reason it never triggers `unclear` there.